app/services/context_service.rb

```diff
@@ -23,6 +32,20 @@ def org_admin_context
     end
   end
 
+  def service_to_service_headers
+    {
+      'x-rh-rbac-psk'       => ENV['RBAC_PSK'],
+      'x-rh-rbac-account'   => account_number,
+      'x-rh-rbac-client-id' => 'approval',
+      'x-rh-identity'       => nil
```

Comment:
@bzwei wouldn't this conflict with 
https://github.com/RedHatInsights/insights-api-common-rails/blob/064511ec143d299c0a3a271dfd0ea24e6f86965d/lib/insights/api/common/rbac/service.rb#L59 we will set the x_rh_identity to nil, not sure if that is not going to get sent over, we might need a compact in this code to remove the nil entries.

Reply:
I tested and it works